### ISSUE TITLE
FP: Incorrect lines count before IF

### ISSUE THREAD

**Issue by CZechBoY** [slevomat/coding-standard#631]
Hi,
php code:
```php
<?php declare(strict_types = 1);

use Ninjify\Nunjuck\Environment;

if (@! include __DIR__ . '/../vendor/autoload.php') {
	echo 'Install Nette Tester using `composer update --dev`';
	exit(1);
}
// Configure environment
Environment::setup(__DIR__);
```
gives me this error:
```
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 5 | ERROR | [x] Expected 1 lines after "if", found 0.
   |       |     (SlevomatCodingStandard.ControlStructures.ControlStructureSpacing.IncorrectLinesCountAfterControlStructure)
```

Is it supported or should I ignore this error?

**Comment by kukulich** [comment]
The list od supported tokens is as default list in the sniff.

The reported error looks valid.

**Comment by CZechBoY** [comment]
AFTER is no blank newline...

**Comment by kukulich** [comment]
@CZechBoY Yes, that is exactly reported.

**Comment by CZechBoY** [comment]
and there is no blank line, so the error is correct and this issue is not issue
